platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/StateConfig.java

```diff
@@ -103,12 +95,9 @@ public record StateConfig(
         @ConfigProperty(defaultValue = "20") int stateSavingQueueSize,
         @ConfigProperty(defaultValue = "900") int saveStatePeriod,
         @ConfigProperty(defaultValue = "5") int signedStateDisk,
-        @ConfigProperty(defaultValue = "false") boolean dumpStateOnAnyISS,
-        @ConfigProperty(defaultValue = "true") boolean dumpStateOnFatal,
         @ConfigProperty(defaultValue = "false") boolean haltOnAnyIss,
         @ConfigProperty(defaultValue = "false") boolean automatedSelfIssRecovery,
         @ConfigProperty(defaultValue = "false") boolean haltOnCatastrophicIss,
-        @ConfigProperty(defaultValue = "21600") long secondsBetweenISSDumps,
```

Comment:
I'd suggest doing a search in the regression repo for these settings. Removing them without cleaning up JRS will result in failed tests.

Reply:
good point, done